### ISSUE TITLE
Components not loading No module named 'landlab.components.flexure.ext.flexure1d'

### ISSUE THREAD

**Issue by pfeiffea** [landlab/landlab#821]
Hi there. I'm working on the developers version of LL. Hadn't updated from default branch in a while. Now when I try to import components, I get the error" No module named 'landlab.components.flexure.ext.flexure1d'"

I tried python `setup.py develop`  (as per @mcflugen 's suggestion ) but got the following message: 
`  File "C:\Users\pfeif\Anaconda3\lib\site-packages\Cython\Compiler\Code.py", line 368, in load
    raise ValueError("More than one filename match found for utility code " + util_code_name)
ValueError: More than one filename match found for utility code Capsule`

**Comment by mcflugen** [comment]
@pfeiffea The initial problem is (almost) definitely that you need to recompile the extension modules. The `flexure_1d` module must have been added since the last time you compiled things.

I'm not quite sure about the second error. Was that the entire error message?

Where is the source code that you're working with? Is it your [landlab fork](https://github.com/pfeiffea/landlab)? Which branch?

**Comment by saisiddu** [comment]
Hey Allison, try looking at these issues: #768 and #773. I am suspecting this is an issue with the latest compilers distributed by Microsoft (I might be totally wrong). https://github.com/landlab/landlab/issues/773. Can you try to install the latest compilers and reinstall Landlab?

**Comment by mcflugen** [comment]
@pfeiffea Are you using Spyder? If so, could you please try installing and running landlab outside of Spyder?

**Comment by mcflugen** [comment]
@pfeiffea I've never seen this before. This is weird. What files to do have in `landlab/ca` that aren't `*.py` files? Cython seems to be finding multiple files where it only expects one. Are there multiple files that start with `cfuncs`?

**Comment by pfeiffea** [comment]
@mcflugen Yes, I'm in Spyder. Not sure what you mean about installing landlab outside of Spyder-- I'm doing this in the Anaconda Prompt. Can you clarify?

I was going through the directions for [Installing Landlab from source code](https://github.com/landlab/landlab/wiki/Installing-Landlab-from-source-code-(%22developer-install%22)), and ran into errors related to the flexure component when running pytest ... but perhaps this doesn't tell us anything new.

**Comment by mcflugen** [comment]
@pfeiffea I was just wanting you to run everything from the Anaconda Prompt and not start up Spyder at all. Some folks were having a similar issue as you when they used Spyder. However, based on the complete error message, it looks like you're having another issue.

I don't think flexure is the source of the problem. I think it's just that it needed to be rebuilt because it had changed recently.

**Comment by pfeiffea** [comment]
Here's what I have in landlab/ca
![image](https://user-images.githubusercontent.com/35848814/47592135-dfb58500-d926-11e8-8112-9d6bc9383aec.png)

**Comment by nathanlyons** [comment]
@mcflugen Do you think uninstalling then reinstalling cython, then `setup.py develop` is worth a shot? See [https://github.com/stan-dev/pystan/issues/468](https://github.com/stan-dev/pystan/issues/468).

**Comment by pfeiffea** [comment]
Brilliant! after
 `conda uninstall cython
conda install cython`

I ran `python setup.py develop` without getting an error message. I can now import components! 

I think this means the problem is solved, though I do still get an error message when I run `pytest`

`(base) C:\Users\pfeif\Documents\GitHub\landlab>pytest
============================= test session starts =============================
platform win32 -- Python 3.6.7, pytest-3.9.1, py-1.7.0, pluggy-0.8.0
rootdir: C:\Users\pfeif\Documents\GitHub\landlab, inifile: setup.cfg
plugins: remotedata-0.3.0, openfiles-0.3.0, doctestplus-0.1.3, arraydiff-0.2
collected 0 items / 1 errors

=================================== ERRORS ====================================
______________________________ ERROR collecting  ______________________________
..\..\..\Anaconda3\lib\site-packages\_pytest\config\__init__.py:388: in _getconftestmodules
    return self._path2confmods[directory]
E   KeyError: local('C:\\Users\\pfeif\\Documents\\GitHub\\landlab\\landlab\\flexure\\tests')

During handling of the above exception, another exception occurred:
..\..\..\Anaconda3\lib\site-packages\_pytest\config\__init__.py:416: in _importconftest
    return self._conftestpath2mod[conftestpath]
E   KeyError: local('C:\\Users\\pfeif\\Documents\\GitHub\\landlab\\landlab\\flexure\\tests\\conftest.py')

During handling of the above exception, another exception occurred:
..\..\..\Anaconda3\lib\site-packages\_pytest\config\__init__.py:422: in _importconftest
    mod = conftestpath.pyimport()
..\..\..\Anaconda3\lib\site-packages\py\_path\local.py:668: in pyimport
    __import__(modname)
landlab\flexure\__init__.py:8: in <module>
    from .flexure import Flexure
landlab\flexure\flexure.py:63: in <module>
    from ...utils.decorators import use_file_name_or_kwds
E   ValueError: attempted relative import beyond top-level package

During handling of the above exception, another exception occurred:
..\..\..\Anaconda3\lib\site-packages\py\_path\common.py:377: in visit
    for x in Visitor(fil, rec, ignore, bf, sort).gen(self):
..\..\..\Anaconda3\lib\site-packages\py\_path\common.py:429: in gen
    for p in self.gen(subdir):
..\..\..\Anaconda3\lib\site-packages\py\_path\common.py:418: in gen
    dirs = self.optsort([p for p in entries
..\..\..\Anaconda3\lib\site-packages\py\_path\common.py:419: in <listcomp>
    if p.check(dir=1) and (rec is None or rec(p))])
..\..\..\Anaconda3\lib\site-packages\_pytest\main.py:568: in _recurse
    ihook = self.gethookproxy(path)
..\..\..\Anaconda3\lib\site-packages\_pytest\main.py:422: in gethookproxy
    my_conftestmodules = pm._getconftestmodules(fspath)
..\..\..\Anaconda3\lib\site-packages\_pytest\config\__init__.py:399: in _getconftestmodules
    mod = self._importconftest(conftestpath)
..\..\..\Anaconda3\lib\site-packages\_pytest\config\__init__.py:439: in _importconftest
    raise ConftestImportFailure(conftestpath, sys.exc_info())
E   _pytest.config.ConftestImportFailure: (local('C:\\Users\\pfeif\\Documents\\GitHub\\landlab\\landlab\\flexure\\tests\\conftest.py'), (<class 'ValueError'>, ValueError('attempted relative import beyond top-level package',), <traceback object at 0x00000203619CC508>))`

**Comment by mcflugen** [comment]
@pfeiffea Are you syncing this branch with an upstream branch? If so, which one? There seems to be some problems with the branch that is also preventing me from running the tests.

**Comment by pfeiffea** [comment]
@mcflugen Well, that's new. There's a chance I messed that up somehow in my earlier attempts to fix this problem. Earlier today I was able to 'update from default branch' in my the Make_network_transport_model branch, but that appears to be disabled(greyed out) now. Any hints? 

In case this helps, `git remote show origin` gives me
![image](https://user-images.githubusercontent.com/35848814/47596104-eb11ac00-d938-11e8-9f04-5bcdc51e8fe3.png)

**Comment by pfeiffea** [comment]
Great, thanks so much for the help with this. I'll get in touch if the
upstream syncing issues persist.
